src/test_all.py

```diff
@@ -127,7 +128,19 @@ def assert_language_name(name):
                     except KeyError:
                         raise Exception(f" * Looks like we have a repeated example here! "
                                         f"Merge outputs before removing duplicates. :-/ \n {instance}")
-
+                        
+                        
+            #Make sure classes are balanced
+            if 'Classification' in data['Categories']:
```

Comment:
I think it's better to infer this automatically (rather than checking for the classification tag in the categories)
Basically, find the set of unique category labels (`set( [x[`output`] for x in data['Instances'][:100] )`) and check it is small (say, less than 15 categories).

Reply:
I'm not sure if I get it correctly; you mean to run this script on all the tasks?

Reply:
If it is not too slow, I think it's worth the shot.